autoagents/data/generate_action_data.py

```diff
@@ -46,4 +47,5 @@ def main(q):
     with open(args.goals, "r") as file:
         data = json.load(file)
     with Pool(processes=4) as pool:
-        pool.map(main, data)
+        for _ in tqdm(pool.imap_unordered(main, data), total=len(data)):
```

Comment:
Probably don't need the `print` on line 38 after using `tqdm` to track progress

Reply:
The `print` statement is removed.